### PR TITLE
@cap-js/audit-logging

### PR DESCRIPTION
**Pull request by sjvans**

### CODE REVIEW

**Comment by sjvans** [comment]
@renejeglinsky i had done XX_*.md to fix the order of chapters here:
Screenshot 2023-08-25 at 08 56 30
we want:
1. Basics
2. PDM
3. DRM
4. Audit Logging

File: menu.md [hunk 1]
```diff
@@ -47,7 +47,9 @@
 - [Data Privacy](guides/data-privacy/)
   - [Basics](guides/data-privacy/introduction)
   - [Personal Data Management](guides/data-privacy/pdm)
-  - [Audit Log](guides/data-privacy/audit-log)
+  - [Data Retention Management](guides/data-privacy/drm)
+  - [Audit Logging](guides/data-privacy/audit-logging)
+  - [Sample App](guides/data-privacy/sample-app)
```
Comment (sjvans) [inline menu.md:52]:
```suggestion
```

File: guides/data-privacy/sample-app.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
```
Comment (sjvans) [inline guides/data-privacy/sample-app.md:1]: TODO: delete this file before merging

File: guides/data-privacy/drm.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+shorty: Data Retention Management
+synopsis: >
+  Use the SAP Data Retention Manager (DRM) with a CAP application.
+breadcrumbs:
+  - Cookbook
+  - Data Privacy
+  - Data Retention
+status: released
```
Comment (renejeglinsky) [inline guides/data-privacy/drm.md:9]:
```suggestion
# status: released
```

**Review by renejeglinsky — APPROVED** [review]
Review to be done later

**Pull request merged**